src/components/PageTitle/index.js

```diff
@@ -4,13 +4,10 @@ import style from './style.css'
 import classNames from 'classnames'
 
 class PageTitle extends Component {
-  componentDidMount() {
-    this.container.focus()
-  }
 
   componentDidUpdate(prevProps) {
-    const { title, subTitle } = this.props
-    if (title !== prevProps.title || subTitle !== prevProps.subTitle) {
+    const { subTitle } = this.props
+    if (subTitle !== prevProps.subTitle) {
```

Comment:
Why removing `title` from this condition?

Reply:
As discussed, this is still needed so have reinstated it.